src/main/kotlin/no/nav/familie/ef/mottak/task/SendSøknadMottattTilDittNavTask.kt

```diff
@@ -23,7 +23,7 @@ class SendSøknadMottattTilDittNavTask(private val producer: DittNavKafkaProduce
         producer.sendToKafka(søknad.fnr,
                              lagLinkMelding(søknad.dokumenttype),
                              task.payload,
-                             task.id.toString(),
+                             task.metadata["eventId"].toString(),
```

Comment:
Slik jeg tolker det så kan vi ende opp med å sende med samma eventId 2 ganger her? En for dokumentasjonsbehov og en for mottatt søknad - er det mulig?

Reply:
Hm, det er det jeg har prøvd å unngå ved opprettelse av task i ArkiverSøknadTask:34, men mulig jeg har oversett noe?